src/test/java/com/znsio/teswiz/steps/VodQASteps.java

```diff
@@ -37,4 +35,13 @@ public void scrollToElement() {
     public void isElementWithTextVisible() {
         new VodqaBL(SAMPLE_TEST_CONTEXT.ME, Runner.getPlatform()).isElementWithTextVisible();
     }
+    @When("I tap in the middle of the screen")
+    public void iTapInTheMiddleOfTheScreen() {
+        new VodqaBL(SAMPLE_TEST_CONTEXT.ME, Runner.getPlatform()).tapInTheMiddleOfTheScreen();
+    }
+
+    @Then("I am able to move from {string} page to next page")
+    public void iAmAbleToMoveFromPageToNextPage(String pageHeader) {
+        new VodqaBL(SAMPLE_TEST_CONTEXT.ME, Runner.getPlatform()).verifyUserOnNextSection(pageHeader);
```

Comment:
it would be better if methods are named in a similar manner
iAmAbleToMoveFromPageToNextPage() and verifyUserOnNextSection() doesn't seem to have much common when reading the names. Write name according to the action that is happening at last in the screen itself. Name all methods (steps, BL, abstract screen, androidscreen) accordingly.